README.md

```diff
@@ -6,7 +6,7 @@ Exercism exercises in F#
 
 ## Support and Discussion
 
-We have a [gitter channel](https://gitter.im/exercism/xfsharp) where you can get support for any issues you might be facing (build setup, failing tests, etc.) or brainstorm with other people for the solution.
+We have an [F# subcategory](https://forum.exercism.org/c/programming/f/67) on the [Exercism forum](https://forum.exercism.org/) where you can get support for any issues you might be facing (build setup, failing tests, etc.) or brainstorm with other people for the solution.
```

Comment:
I wasn't sure if I should just delete this bit instead of replacing it. We don't expect people to be looking at the readme if they need help solving exercises.

If this is for people who are helping develop the track, then we should direct them to the Building Exercism category: https://forum.exercism.org/c/exercism/building-exercism/125